draft-voit-rats-attestation-results.md

```diff
@@ -200,18 +200,28 @@ Additionally, the Relying Party must have confidence that the Trustworthiness Cl
 
 ### Attesting Environment
 
-For the Attesting Environment identity, there MUST exist a chain of trust ultimately bound to a hardware-based root of trust in the Attesting Environment.
+This document recognizes three general categories of Attesting Environments.
+
+1. TPM-based: A cryptoprocessor exists which continually hashes security measurements in a way which prevents an Attester from lying about measurements taken since boot (e.g., TPM2.0.)
```

Comment:
Maybe we should call it "Secure Element"-based rather than TPM? We wouldn't know where to put vTPM-based Attesting Environments (AE) otherwise (TPM-based or Process-based?)

Reply:
Secure Element based works for me.

Reply:
I'd hold-off on the Secure Element choice for now, because I am under impression that eSE, TPM, etc. are all a bit loaded terms. I'd prefer to see Hardware Security Module (HSM), as a more neutral term.

Reply:
HSM based?    Honestly I wouldn't use a vTPM alone for Security Measurements, unless it was in a Secure Enclave.  I would however trust a vTPM for Secure Storage if it was bound to an HSM.

Reply:
> HSM based- Is that ok with everyone?

In my experience an HSM is a big blue box on a LAN, whereas the SEs that I know have a more "chip-py" form factor.

In case this is not a shared perspective, I will happily adjust my mental pictures catalogue and go with HSM :-)

Reply:
[now commenting in the right thread, thx for always correcting me 😊]

I checked with the Fraunhofer competency cluster and got this back as a counter example that aligns with Thomas' point.

https://cpl.thalesgroup.com/encryption/hardware-security-modules

The Internet seems to disagree with itself here. Which reduces the viability of "HSM" here, a bit. Which... also does not make this easier.

If it is neither e(SE) nor HSM, what is it? Any other thoughts?

Reply:
> If it is neither e(SE) nor HSM, what is it? Any other thoughts?

I'm happy to go with HSM until we find a better term.